tools/run_tests/dockerize/docker_run_tests.sh

```diff
@@ -18,6 +18,7 @@
 
 set -e
 
+# shellcheck disable=SC2154
```

Comment:
Did you try changing the next line to `export CONFIG=${config:?}` instead of disabling shellcheck? See https://github.com/koalaman/shellcheck/wiki/SC2154. SC2154 complains about uninitialized variables, which seems pretty reasonable here.

Reply:
Seems in our testing infrastructure, the `$config` is not always set or non-null. I guess I need to put it back.

Reply:
You could not export it if it's empty or unset. (it may also be reasonable to assign it a default of opt, as done in e.g https://github.com/grpc/grpc/blob/master/tools/run_tests/helper_scripts/build_php.sh, but that would require confirming empty=opt)

Reply:
The test is passed. PTAL.